<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roposition</class>
 <widget class="QWidget" name="roposition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_show_option" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>500</height>
    </rect>
   </property>
   <widget class="QWidget" name="dir_widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>327</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	border-width:2px;
	border-style:solid;
	border-color:black;
	margin-top:0.5ex;
}</string>
    </property>
    <widget class="QPushButton" name="btn_2_5_up">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>40</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resource.qrc">
       <normaloff>:/images/bin/resource/images/dir_up.png</normaloff>:/images/bin/resource/images/dir_up.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>27</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_1_4_left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>40</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resource.qrc">
       <normaloff>:/images/bin/resource/images/dir_left.png</normaloff>:/images/bin/resource/images/dir_left.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>27</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_1_4_right">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>40</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resource.qrc">
       <normaloff>:/images/bin/resource/images/dir_right.png</normaloff>:/images/bin/resource/images/dir_right.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>27</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2_5_down">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>40</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resource.qrc">
       <normaloff>:/images/bin/resource/images/dir_down.png</normaloff>:/images/bin/resource/images/dir_down.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>27</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_6_right">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>40</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resource.qrc">
       <normaloff>:/images/bin/resource/images/clockwise.png</normaloff>:/images/bin/resource/images/clockwise.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>27</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_6_left">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>40</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resource.qrc">
       <normaloff>:/images/bin/resource/images/counterclockwise.png</normaloff>:/images/bin/resource/images/counterclockwise.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>27</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_pos">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>17</y>
      <width>281</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border-width:2px;
	border-style:solid;
	border-color:black;
	margin-top:0.5ex;
}

QGroupBox::title{
	subcontrol-origin:margin;
	subcontrol-position:top left;
	left:10px;
	margin-left:0px;
	padding:0.1px;
}</string>
    </property>
    <property name="title">
     <string>位置</string>
    </property>
    <widget class="QLabel" name="label_1_x_data">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>21</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3_z_data">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>73</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6_q3_data">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2_y_data">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>47</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4_q1_data">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>98</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5_q2_data">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>124</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_1_x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>25</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2_y">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>47</y>
       <width>25</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3_z">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>73</y>
       <width>25</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5_q2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>124</y>
       <width>25</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4_q1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>98</y>
       <width>25</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6_q3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>25</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_form">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>241</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>位置格式...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7_q4_data">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>176</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7_q4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>176</y>
       <width>25</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>q4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7_q4_unit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>176</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4_q1_unit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>98</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6_q3_unit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2_y_unit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>47</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5_q2_unit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>124</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3_z_unit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>73</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_1_x_unit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>21</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>394</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>运动</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_option">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>17</y>
      <width>331</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border-width:2px;
	border-style:solid;
	border-color:black;
	margin-top:0.5ex;
}

QGroupBox::title{
	subcontrol-origin:margin;
	subcontrol-position:top left;
	left:10px;
	margin-left:0px;
	padding:0.1px;
}</string>
    </property>
    <property name="title">
     <string>选项</string>
    </property>
    <widget class="QPushButton" name="btn_1_coord">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>35</y>
       <width>94</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>基坐标</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2_setaxis">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>73</y>
       <width>94</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>轴1-3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_setaxisspeed">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>111</y>
       <width>94</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>50%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_4_tool">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>149</y>
       <width>94</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_5_usr">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>187</y>
       <width>94</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>41</y>
       <width>114</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>坐标系:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>232</y>
       <width>114</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>操作杆锁定：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>79</y>
       <width>114</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设置轴:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>117</y>
       <width>114</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设置轴速度:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>270</y>
       <width>114</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>增量:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>155</y>
       <width>114</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>工具坐标:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>193</y>
       <width>114</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>工件坐标:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_6_lock">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>226</y>
       <width>94</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>无</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_7_incr">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>264</y>
       <width>94</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>无</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>347</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前模式:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>440</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>报警</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_moving">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>397</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_dir">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>287</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border-width:2px;
	border-style:solid;
	border-color:black;
	margin-top:0.5ex;
}

QGroupBox::title{
	subcontrol-origin:margin;
	subcontrol-position:top left;
	left:10px;
	margin-left:0px;
	padding:0.1px;
}</string>
    </property>
    <property name="title">
     <string>操作杆方向</string>
    </property>
    <widget class="QLabel" name="label_images_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>76</width>
       <height>76</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_images_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>76</width>
       <height>76</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_images_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>76</width>
       <height>76</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_ch_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ch_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ch_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_alarm">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>443</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_model">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
